src/components/Dropdown/Dropdown.tsx

```diff
@@ -1,12 +1,16 @@
 import { createElement, isValidElement, KeyboardEvent, PropsWithChildren, ReactNode, useEffect, useMemo, useRef, useState } from 'react';
 import { DropdownContext, DropdownContextType } from './DropdownContext';
 import { InputContext, InputContextType } from './InputContext';
-import useRootClose from '@restart/ui/useRootClose';
+import useRootClosePkg from '@restart/ui/useRootClose';
+const useRootClose = typeof useRootClosePkg === 'function' ? useRootClosePkg : useRootClosePkg['default'];
```

Comment:
yikes we have to do this in our source code to make vite happy? cc @mkilpatrick

Reply:
This was the only way I was able to make typescript, vite, and deno happy at the same time. Potentially there is a config option or plugin that could handle this during the build step, but I wasn't able to find one. tsup has a cjsInterop flag but that doesn't fix this issue